examples/dataset/rain_forecast/README.md

```diff
@@ -38,3 +38,30 @@ optional arguments:
   --model {ann,logreg}  Name of model to test.
   --dataset DATASET     Name of dataset to use for testing.
 ```
+
+## Quality Standards Guide
+
+Once the dataset and results have been uploaded to Kolena, visit [Kolena](https://app.kolena.io/redirect/) to
+test the rain forecast models. See our [QuickStart](https://docs.kolena.io/dataset/quickstart/) guide
```

Comment:
What do you mean by "to test the models"?

Reply:
test models on the set quality standards — happy to change if you have a suggestion.